### PR TITLE
Add SlimHUD.app v1.3.6

### PR DESCRIPTION
**Pull request by ans87gh**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by ran-dall** [comment]
@ans87gh Have you checked if this requires a `zap` stanza?

**Comment by ans87gh** [comment]
@ran-dall Sorry no I didn't. I don't know how to do that.

**Comment by ran-dall** [comment]
@ans87gh You can use GitHub Desktop to pull down your PR. From there I would install it, and then run @nrlquaker's createzap; so `$brew cask createzap <cask>`. That should automatically create the `zap` stanza for you. Otherwise, you can fish out the files paths from AppCleaner or another uninstaller and make the `zap` stanza by hand, it's not hard.

**Comment by ans87gh** [comment]
@ran-dall that's ok, and the offer is more than plenty. I'm just not sure what you meant by install it. I am assuming you mean to pull my PR, install the app from the PR, and then run brew cask createzap and take the output and add it to the PR. Am I understanding it correctly?

**Comment by ran-dall** [comment]
@ans87gh Yup, you got it.

**Comment by ran-dall** [comment]
@ans87gh Also you should note that sometimes it doesn't work and using an uninstaller is the only way to make that `zap` stanza.

**Comment by ans87gh** [comment]
@ran-dall no problem. I also have iTrash and CCM so both can find the same files. I just want to ask if I can do the same with Tower.

**Comment by ran-dall** [comment]
@ans87gh Probably, I use Tower for everything as well; however for pulling down other people's PRs, GitHub Desktop is better. So for this, you could probably use whichever one you'd prefer.

**Comment by ans87gh** [comment]
@ran-dall now after finding slimhud-branch under the remotes, I right click and select branch. But then what. I search for the formula in terminal to install, but its not there. Or do I run the install from someplace else?

**Comment by ran-dall** [comment]
@ans87gh Did you check out the branch?

**Comment by ans87gh** [comment]
@ran-dall I don't think I did; I was just playing with anything that allowed me to pull the branch and tested to see if I had local access via cli to install the app with brew cask install

**Comment by ran-dall** [comment]
@ans87gh You need to check out the PR to have the cask formula available to you.

**Comment by ran-dall** [comment]
@ans87gh Otherwise, you could just download it manually. However if you go this route, you'll need to make the `zap` stanza manually.

**Comment by ans87gh** [comment]
@ran-dall I assumed as much about the manual part. And thanks about adding the check out part. I'll do that now.

**Comment by ans87gh** [comment]
@ran-dall So I checked it out, it is now the head under branches. But not being found via brew search

**Comment by ran-dall** [comment]
@ans87gh I'm not sure, it's kinda hard to follow without seeing what you're doing. That said, I'd recommend you do it manually if you can't figure out how to work it correctly.

**Comment by ans87gh** [comment]
@ran-dall will do

**Comment by ran-dall** [comment]
@ans87gh If you need any help, just ping me. I need to step out soon, but I'll check when I get back.

**Comment by ans87gh** [comment]
@ran-dall did my own workaround. Recreated the cask, installed, ran the app, and now adding the zap stanza info.

**Review by ran-dall — APPROVED** [review]
